### DISCUSSION TITLE
RPC Events abfangen wie VALVE_STATE, VALVE_ADAPTION und FROST_PROTECTION

### DISCUSSION THREAD

**Discussion by AzonInc** [SukramJ/aiohomematic#943] [Q&A] [answered]
Was spricht eigentlich dagegen mehr RPC Events an Home Assistant weiterzuleiten?
Ich habe bisher keine andere Möglichkeit als NodeRED gefunden, um diese Events auszuwerten.

Wäre es möglich das auch in diesem Projekt zu realisieren?

Mein Bruder hatte mal sowas in Nodered erstellt:
![image](https://user-images.githubusercontent.com/11911587/217675752-679bf8e9-1dd8-4212-9392-3f23d319e5db.png)

Nun wollten wir das am liebsten in Home Assistant abbilden.

**Comment by SukramJ** [comment, marked as answer]
Das geht mit [unignore](https://github.com/danielperna84/custom_homematic#unignore-device-parameters).

**Reply by AzonInc** [reply]
Sehr schön, vielen lieben Dank.
Ich probiere es nachher gleich Mal aus.

Ich hatte das zwar vorher schonmal in der Doku gelesen aber anders verstanden.

**Reply by AzonInc** [reply]
Also ich hab das ganze mal getestst und ich bekomme auf jeden Fall die Daten die ich benötige.
Allerdings leider nicht als Event.

Es wäre praktisch wenn man diese als Events bekäme, damit gleich der auslösende Aktor als Entity dabei ist und man nicht alle entities in der automation einbinden muss. So wie bei den UNREACH Events wäre es wesentlich einfacher.

Ist es denn allgemein unerwünscht mehr Events zu senden oder könnte man den Code diesbezüglich erweitern?

**Reply by SukramJ** [reply]
Da alle Änderungen in HA als Event gesendet werden musst Du nur auf `state_changed` lauschen.

**Reply by AzonInc** [reply]
Genau, aber wenn ich jetzt eine Automation habe die bei hunderten von state changes in kurzer zeit die ganze Zeit zuhört aud ausgeführt wird (auch wenn danach eine bedingung kommt) verursacht das doch aber wesentlich mehr CPU Last oder nicht?

Ein spezifisches Event wäre da doch wesentlich effizienter was die Systemlast angeht.

**Reply by SukramJ** [reply]
Filtern?
